kickstarts/centos8_build_mahcine.ks

```diff
@@ -0,0 +1,143 @@
+#version=RHEL8
```

Comment:
File name typo - `centos8_build_mahcine` -> `centos8_build_machine`  (Also in the readme)